### DISCUSSION TITLE
Duplicated page's title and heading H1

### DISCUSSION THREAD

**Discussion by h7b** [ObsidianPublisher/obsidian-github-publisher#128] [Q&A — Template] [answered]
Hi Mara-Li,

When I used your [netlify template](https://github.com/ObsidianPublisher/publisher-template-netlify), in the published page, the page's title and its heading H1 are always duplicated (photo below)

![Screenshot 2023-03-29 233619](https://user-images.githubusercontent.com/86811584/228673430-709c3bb6-90e3-42bb-834d-e3d0e7075db3.jpg)

What should I do to have the same behavior as your [plugin's documentation website](https://obsidian-publisher.netlify.app/mkdocs%20template/code%20of%20conduct/) (photo below). That is: the published page displayed only its heading H1, and the "title" key in frontmatter is used for the `nav` section.

![Screenshot 2023-03-29 233540](https://user-images.githubusercontent.com/86811584/228673471-e2d3ce84-bc59-40c6-b60a-7144d27c6509.jpg)

## What did I try?

- I put in the frontmatter of each note, a key "title"
- I text-diff the `mkdocs.yml` in my repo and your docs page repo, but did not have the idea which option leads to the difference

**Comment by Mara-Li** [comment]
Hello!
In fact it's a normal behavior of mkdocs/material mkdocs.

I not pretty sure the combo (can't check at the moment) but Mkdocs will always add the file title in h1 if there is no h1.

Here is the raw code of your screenshoted file: 

![image](https://user-images.githubusercontent.com/30244939/228674830-65e25d2e-3e49-4e4c-a74f-1bf51c58d0b1.png)
(you can check the source of the docs here : https://github.com/ObsidianPublisher/obsidian-mkdocs-publisher-docs/blob/main/docs/Mkdocs%20Template/Code%20of%20conduct.md?plain=1)

An future idea on my side is to remove the first h1 if it's the same of the title.
Maybe using js or during build?

**Reply by h7b** [reply]
Yes, I also checked the raw content of that page from the docs repo as you pointed out.

I also see that in your docs, this [page](https://obsidian-publisher.netlify.app/github%20publisher/settings/embed/) also has duplicated title and heading. This page has "title" key and no H1 heading. So as you said, this might be the expected behavior from mkdocs.

I will research more from mkdocs-material document

**Reply by Mara-Li** [reply]
I think I will write something in the override template or hook to prevent that : if there is no H1 in the file, don't add one. 

Need to check how to that...

**Comment by Mara-Li** [comment, marked as answer]
Hello!
In fact it was pretty simple 🍰 

It was something added by material-template in `overrides/partials/content.html`. 

I updated the [follow template](https://github.com/ObsidianPublisher/follow_template) : 
1. Download the new [`overrides/partials/content.html`](https://github.com/ObsidianPublisher/follow_template/blob/main/overrides/partials/content.html). 
2. Add it to your overrides folder 
3. Set, in your `mkdocs.yml` the new key in `extra` : `no-auto-h1` to true (https://github.com/ObsidianPublisher/follow_template/blob/4a62f3bd12a007b3bf4020b1402a34f4b47e9e63/mkdocs.yml#L128)
